src/SonarScanner.MSBuild.PreProcessor/Roslyn/Model/Plugin.cs

```diff
@@ -23,10 +23,17 @@
 namespace SonarScanner.MSBuild.PreProcessor.Roslyn;
 
 /// <summary>
-/// Data class for a single SonarQube plugin containing an analyzer
+/// Data class for a single SonarQube plugin containing an analyzer.
 /// </summary>
 public class Plugin
 {
+    /// <summary>
```

Comment:
What's the plan with ruleNamespace and analyzerId? As we've done bunch of parsing, it should all come out of that effort.

There should be a ticket

Reply:
Please update the Epic to document those two oversights